### ISSUE TITLE
Hide forecast UserMessage when week is confirmed

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#651]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
The 'forecasted hours' UserMessage is unnecessary after the week has been confirmed.

![image](https://user-images.githubusercontent.com/7300548/98658398-473ea180-2343-11eb-9c19-b7f6258535c9.png)

### Description
* When week is confirmed, the forecast is no longer useful (with the exception of forecasting accuracy statistics which can be handled on the backend)
* The user then no longer has any need for this metric, so it just clutters the UI
* Basically, when periodConfirmed == true, forecast-UserMessage should not be rendered
